### PR TITLE
HPCC-26680 ESDL tool not honoring exceptions_inline

### PR DESCRIPTION
**Pull request by asselitx**
The esdl command-line tool and ESDL engine-based services do not honor the
exceptions_inline attribute when creating XSD or WSDL output.

Update the esdl transform to honor the exceptions_inline attribute in the
same way that hidl-based services do- when it appears in the EspResponse
element of an ecm file.

Do not change DESDL service wsdl/xsd generation to maintain compatibility
with current DESDL service behavior. This is addressed in HPCC-27342.

Ensure the esdl tool generated wsdl validates by defining the wsdl namespace
prefix it uses.

Signed-off-by: Terrence Asselin <terrence.asselin@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [x] I have checked that this change does not introduce memory leaks.
  - [x] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [x] Performance
  - [x] Security
  - [x] Thread-safety
  - [x] Cloud-compatibility
  - [x] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by rpastrana** [comment]
@asselitx can you provide a sample wsdl generated w/ your recent changes, thanks.

File: esp/xslt/esxdl2xsd.xslt [hunk 1]
```diff
@@ -396,7 +400,9 @@
                 <wsdl:operation>
                     <xsl:attribute name="name"><xsl:value-of select="@name"/></xsl:attribute>
                     <soap:operation style="document">
-                        <xsl:attribute name="soapAction"><xsl:value-of select="$serviceName"/>/<xsl:value-of select="@name"/>?ver_=<xsl:value-of select="$version"/></xsl:attribute>
+                        <xsl:attribute name="soapAction">
+                            <xsl:value-of select="$serviceName"/>/<xsl:value-of select="@name"/>?ver_=<xsl:value-of select="number($version)"/>
```
Comment (rpastrana) [inline esp/xslt/esxdl2xsd.xslt:404]: this seems like a worthwhile change, but it's outside the scope of this change

**Review by rpastrana — CHANGES_REQUESTED** [review]
@asselitx change looks good. 
However the version number change should be in it's own dedicated PR, mostly because we're not clear on the impact of losing the floating point.
Also, the test files seem ok, but only skimmed over them.

**Comment by asselitx** [comment]
@timothyklemm  in Rodrigo's absence could you please review?

**Review by rpastrana — APPROVED** [review]
@asselitx looks reasonable.

**Comment by richardkchapman** [comment]
@asselitx I think the checks failed to run because github was having issues - if you force-push an amended commit (without any changes) then it should retrigger them

**Comment by asselitx** [comment]
Note just pushed the last commit to re-trigger the workflow checks to run. It didn't contain any changes, just a rebase and force push.

**Pull request merged**